pkg/gofr/handler_test.go

```diff
@@ -0,0 +1,79 @@
+package gofr
```

Comment:
test desc should be added in all assert statements
"TEST[%d], Failed.\n%s", i, tc.desc